src/entities/staff/staff.ts

```diff
@@ -75,6 +77,7 @@ export class Staff extends UniverseEntity<StaffPayload, StaffRawPayload> {
   public roles?: StaffRawPayload['roles']
   public permissions?: StaffRawPayload['permissions']
   public invite?: StaffRawPayload['invite']
+  public customId?: StaffRawPayload['custom_id']
```

Comment:
nit: should likely be StaffPayload['custom_id'], but most lines above have the same problem